modules/remove.js

```diff
@@ -0,0 +1,8 @@
+// removes first element having the condition
```

Comment:
```suggestion
import cb from './_cb.js';

// removes first element having the condition
```

This is required for my next suggestion...